python/dgl/nn/pytorch/explain/subgraphxexplainer.py

```diff
@@ -0,0 +1,525 @@
+"""Torch Module for SubgraphX"""
+import math
+import torch
+import numpy as np
+from torch import nn
+import networkx as nx
+import dgl
+
+__all__ = ["SubgraphXExplainer"]
+
+def neighbors(node, graph):
```

Comment:
I don't think you need a dedicated function for this since it's only used once.

Reply:
done.